### DISCUSSION TITLE
How to local quickstart (readme is outdated or incomplete) ?

### DISCUSSION THREAD

**Discussion by astralski** [mckaywrigley/chatbot-ui#1845] [Help]
I'm trying to run this project on local. 

at http://localhost:3000/ i got :

<img width="538" alt="image" src="https://github.com/user-attachments/assets/d4ddd673-cc0f-4cb5-95e7-a669c70f62ad">

There's no info at readme how to log in. I guess i have to create a new user at superbase admin panel. But how? 

I tried many times with no effect

<img width="449" alt="image" src="https://github.com/user-attachments/assets/a3dbaf64-3693-4448-aabc-bc48e2ce625d">

**Comment by bernie43** [comment]
You should be able to create a user by clicking Sign Up. 
Seems like the backend isn‘t working. Check if your local Supabase instance is running running properly.
